hadoop-hdds/docs/themes/ozonedoc/static/swagger-resources/recon-api.yaml

```diff
@@ -0,0 +1,1584 @@
+    description: APIs to fetch metadata for the blocks available. **Admin Only**
+  - name: Namespace Metadata
+    description: APIs to fetch metadata for the namespace. **Admin Only**
+  - name: Cluster State
+    description: APIs to fetch data about the cluster state
+  - name: Datanodes
+    description: APIs to fetch data about the Datanodes
+  - name: Pipelines
+    description: APIs to fetch data about the Pipelines
+  - name: Tasks
+    description: APIs to fetch data about status of Recon Tasks
+  - name: Utilization
+    description: APIs to fetch data about space utilization
+  - name: Metrics
+    description: APIs to fetch data about various metrics from Prometheus
+    externalDocs:
+      description: Prometheus API docs
+      url: https://prometheus.io/docs/prometheus/latest/querying/api/
+paths:
+  /containers:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get all Container Metadata information
+      operationId: getContainerInfo
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ContainerMetadata'
+  /containers/{containerId}/keys:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get the Key metadata for all keys present in the given container ID
+      operationId: getContainerKeyMetadata
+      parameters:
+        - name: containerId
+          in: path
+          description: ID of the container for which we want Key Metadata
+          required: true
+          schema:
+            type: integer
+        - name: prevKey
+          in: query
+          description: Only return keys that are present after the given key prevKey prefix
+          required: false
+          schema:
+            type: string
+            example: /vol1/bucket1/key1
+        - name: limit
+          in: query
+          description: Limit of the number of results returned
+          required: false
+          schema:
+            type: integer
+            default: 1000
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/KeyMetadata'
+  /containers/missing:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get the MissingContainerMetadata for all missing containers
+      operationId: getMissingContainers
+      parameters:
+        - name: limit
+          in: query
+          description: Limit of the number of results returned
+          required: false
+          schema:
+            type: integer
+            default: 1000
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MissingContainerMetadata'
+  /containers/{containerId}/replicaHistory:
```

Comment:
Correct the `containerId` param here

Reply:
Please check the reply at https://github.com/apache/ozone/pull/5509#discussion_r1381907219 for the same

Reply:
> Please check the reply at [#5509 (comment)](https://github.com/apache/ozone/pull/5509#discussion_r1381907219) for the same

Pls check above comments and similar for below.

Reply:
Addressed in the latest commit